data_steward/cdr_cleaner/manual_cleaning_rules/remediate_basics.py

```diff
@@ -182,22 +195,48 @@
 SELECT
     i.person_id,
     i.src_id,
-    sb.state_of_residence_concept_id,
-    sb.state_of_residence_source_value,    
+    sb.state_of_residence_concept_id, -- This might be NULL if the person is new from the incremental dataset --
+    sb.state_of_residence_source_value, -- This might be NULL if the person is new from the incremental dataset -- 
     i.sex_at_birth_concept_id,
     i.sex_at_birth_source_concept_id,
     i.sex_at_birth_source_value
 FROM `{{project}}.{{incremental_dataset}}.{{table}}` i
-JOIN `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` sb
+LEFT JOIN `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` sb
 ON i.person_id = sb.person_id
+{% if exclude_lookup_dataset and exclude_lookup_table %}
+WHERE i.person_id NOT IN (
+    SELECT 
+        {% if is_deid %} research_id {% else %} person_id {% endif %}
+    FROM `{{project}}.{{exclude_lookup_dataset}}.{{exclude_lookup_table}}` 
+)
+{% endif %}
 """)
 
 GENERIC_INSERT = JINJA_ENV.from_string("""
 INSERT INTO `{{project}}.{{dataset}}.{{table}}`
 SELECT * FROM `{{project}}.{{incremental_dataset}}.{{table}}`
+{% if exclude_lookup_dataset and exclude_lookup_table %}
+WHERE person_id NOT IN (
+    SELECT 
+        {% if is_deid %} research_id {% else %} person_id {% endif %}
+    FROM `{{project}}.{{exclude_lookup_dataset}}.{{exclude_lookup_table}}` 
+)
+{% endif %}
+""")
+
+GENERIC_INSERT_MAP_EXT = JINJA_ENV.from_string("""
+INSERT INTO `{{project}}.{{dataset}}.{{table}}`
+SELECT * FROM `{{project}}.{{incremental_dataset}}.{{table}}`
+{% if exclude_lookup_dataset and exclude_lookup_table %}
 WHERE {{domain}}_id IN (
```

Comment:
Hi, @hiro-mishima - similar to above question, did you have a specific scenario in mind where this could break?  Otherwise, perhaps we could remove the outer WHERE clause on 231-232.

Reply:
Same as above, mapping tables and ext tables do not have `person_id` columns so these two where clauses are necessary.